.npmignore

```diff
@@ -3,3 +3,6 @@
 .editorconfig
 images/
 test/
+examples/
```

Comment:
See the comments on the PR. Can't exclude this folder directly because it breaks functionality. Other options would work though.